examples/brandai/README.md

```diff
@@ -0,0 +1,46 @@
+# React-WinJS Component Documentation (see it live [here](https://brandai.com/styleguide/WinJS))
+
+This folder contains configuration required to build documentation for react-winjs using [Brand AI](https://brand.ai).
+
+## Flow
+### Build
+Use gulp to build a distribution folder containing:
+* library version of react-winjs
+* WinJS core scripts and stylesheets.
+
+See [webpack.config.json](https://github.com/winjs/react-winjs/blob/master/examples/brandai/webpack.config.js) and [gulpfile.json](https://github.com/winjs/react-winjs/blob/master/examples/brandai/gulpfile.js) for more info on that.
+
+```sh
+$ npm install
+$ gulp build
+```
+
+### Deploy
+Deploying to Brand.ai is done through [brandai-tools](https://github.com/brandai/brandai-tools).
+```sh
+$ npm install -g brandai-tools
+$ npm login
+$ npm deploy
```

Comment:
There's a bug with `brandai deploy` on Windows. Here's part of the stack:

```js
Unhandled rejection SyntaxError: Invalid regular expression: /^dist\/: \ at end of pattern
    at new RegExp (native)
    at \npm\node_modules\brandai-tools\lib\commands\deploy\dist.js:10:21
```

This line in `dist.js` is the problem:

`var pathRegex = new RegExp('^' + path + pathDelimeter);`

On Windows, it resolves to

`var pathRegex = new RegExp("^dist\\");`